File: beginner/python/1018.py
n = int(input())

n100 = n / 100
n1resto = n % 100
n50 = n1resto / 50
n2resto = n1resto % 50
n20 = n2resto / 20
n3resto = n2resto % 20
n10 = n3resto / 10
n4resto = n3resto % 10
n5 = n4resto / 5
n5resto = n4resto % 5
n2 = n5resto / 2
n6resto = n5resto % 2
n1 = n6resto / 1

print('%d\n%d nota(s) de R$ 100,00\n%d nota(s) de R$ 50,00\n%d nota(s) de R$ 20,00\n%d nota(s) de R$ 10,00\n%d nota(s) de R$ 5,00\n%d nota(s) de R$ 2,00\n%d nota(s) de R$ 1,00' % (n, n100, n50, n20, n10, n5, n2, n1)) 

File: beginner/javascript/1018.js
var input = require('fs').readFileSync('stdin', 'utf8');

var valores = input.split("\n");

var n = valores[0];

var n100 = parseInt(n / 100);
var n1resto = parseInt(n % 100);
var n50 = parseInt(n1resto / 50);
var n2resto = parseInt(n1resto % 50);
var n20 = parseInt(n2resto / 20);
var n3resto = parseInt(n2resto % 20);
var n10 = parseInt(n3resto / 10);
var n4resto = parseInt(n3resto % 10);
var n5 = parseInt(n4resto / 5);
var n5resto = parseInt(n4resto % 5);
var n2 = parseInt(n5resto / 2);
var n6resto = parseInt(n5resto % 2);
var n1 = parseInt(n6resto / 1);

console.log(n);
console.log(n100 + " nota(s) de R$ 100,00\n" +
            n50 + " nota(s) de R$ 50,00\n" +
            n20 + " nota(s) de R$ 20,00\n" +
            n10 + " nota(s) de R$ 10,00\n" + 
            n5 + " nota(s) de R$ 5,00\n" + 
            n2 + " nota(s) de R$ 2,00\n" + 
            n1 + " nota(s) de R$ 1,00"); 